app/bundles/LeadBundle/Entity/LeadFieldRepository.php

```diff
@@ -381,4 +381,18 @@ public function getFieldsByType($type)
     {
         return $this->findBy(['type' => $type]);
     }
+
+    /**
+     * @return mixed[]
+     */
+    public function getFieldSchemaData(string $object): array
+    {
+        return $this->_em->createQueryBuilder()
+            ->select('f.alias, f.label, f.type, f.isUniqueIdentifer, f.charLengthLimit')
```

Comment:
Looks like charLengthLimit  columns is from another PR what is not pushed to community yet.

Reply:
Good catch! Thanks @kuzmany!